perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -214,6 +214,44 @@ sub pay : Path('pay') : Args(0) {
     }
 }
 
+# redirect from cc processing - bulky goods only at present
+sub pay_cancel : Local : Args(2) {
+    my ($self, $c, $id, $token) = @_;
+
+    my $property_id = $c->get_param('property_id');
+
+    $c->forward('check_payment_redirect_id', [ $id, $token ]);
+    my $p = $c->stash->{report};
+
+    $c->forward('/auth/get_csrf_token');
+
+    # XXX Photos not present in report as yet
+    my $saved_data = {
+        "chosen_date" => $p->get_extra_field_value('DATE'),
+        "location" => $p->get_extra_field_value('CREW NOTES'),
+        #"location_photo_fileid" => '???',
+        "name" => $p->name,
+        "email" => $p->user->email,
+        "phone" => $p->user->phone,
+        "resident" => "Yes",
+    };
+    my @fields = grep { $_->{name} =~ /ITEM_/ } @{$p->get_extra_fields};
```

Comment:
Should probably ignore any with an empty `value` otherwise the template lists them all (template also expects `cobrand.bulky_items_maximum`, not sure why that's not working):

<img width="629" alt="image" src="https://user-images.githubusercontent.com/4776/205154791-9ccf0ef1-52af-4751-a20b-afd614543195.png">

Reply:
Ah this overlaps with my other PR for fixing the JS photo thing, which will change this so that this will work okay because it won't need the hasher of item_NUM any more.